data/json/items/tool/fishing.json

```diff
@@ -118,8 +118,7 @@
     "symbol": "/",
     "color": "black",
     "use_action": [ "FISH_ROD" ],
-    "qualities": [ [ "FISHING", 1 ] ],
-    "flags": [ "FISH_GOOD" ]
+    "qualities": [ [ "FISHING", 1 ] ]
```

Comment:
It appears that while FISH_GOOD got translated to FISHING 2 in another tool, here FISH_GOOD would translate to FISHING 1. This looks to be an oversight that took place when this tool was initially implemented if I had to take a guess, but I suspect it's not intended to be like this with the current changes?

Reply:
The discrepancy between the two here was likely to have been an oversight initially, but keeping this at FISHING 1 in this PR is deliberate. Reasoning as follows:
- The descriptions for both `hoboreel` and `plastichoboreel` indicate that they are "not as effective as a normal fishing pole", so they have to be inferior to the commercial product `fishing_rod_professional`
- The mechanical operation of the wooden hobo reel and its plastic commercial counterpart are so simple that the commercial version shouldn't confer much advantage in terms of how well it will catch fish. It's a spool for the line integrated to a fixed handle, simple as.
- By comparison, with the professional rod what comes to mind is metal or graphite or carbon fiber construction, spinning reel with hand crank; even the way the rod bends is by design.

Reply:
Ah, that is fair. I'll be perfectly honest, I did not check what item this was - I simply noticed the code inconsistency and figured it's better to point it out than not to. If these are both hoboreels then I do agree that they should probably not differ a whole lot if at all. 

Either way, good job cleaning this all up. Thanks for this!